content/pt-br/docs/reference/access-authn-authz/authorization.md

```diff
@@ -0,0 +1,248 @@
+
+<!-- body -->
+
+No Kubernetes, você deve estar autenticado (conectado) antes que sua requisição possa ser
+autorizada (permissão concedida para acesso). Para obter informações sobre autenticação,
+visite [Controlando Acesso à API do Kubernetes](/pt-br/docs/concepts/security/controlling-access/).
+
+O Kubernetes espera atributos que são comuns a requisições de APIs REST. Isto significa
+que autorização no Kubernetes funciona com sistemas de controle de acesso a nível de organizações
+ou de provedores de nuvem que possam lidar com outras APIs além das APIs do Kubernetes.
+
+## Determinar se uma requisição é permitida ou negada
+
+Kubernetes autoriza requisições de API utilizando o servidor de API. Ele avalia
+todos os atributos de uma requisição em relação a todas as políticas disponíveis a permite ou nega a requisição.
+Todas as partes de uma requisição de API deve ser permitido por alguma política para que possa prosseguir.
+Isto significa que permissões sao negadas por padrão.
+
+(Embora o Kubernetes use o servidor de API, controles de acesso e políticas que
+dependem de campos específicos de tipos específicos de objetos são tratados pelo Admission
+Controller.)
+
+Quando múltiplos modules de autorização são configurados, cada um será verificado em sequência.
+Se qualquer dos autorizadores aprovarem ou negarem uma requisição, a decisão é imediatamente
+retornada e nenhum outro autorizador é consultado. Se todos os módulos de autorização não tiverem
+nenhuma opinião sobre requisição, então a requisição é negada. Uma negação retorna um
+código de status HTTP 403.
+
+## Revisão de atributos de sua requisição
+
+O Kubernetes revisa somente os seguintes atributos de uma requisição de API:
+
+ * **user** - O string de `user` fornecido durante a autenticação.
+ * **group** - A lista de nomes de grupos aos quais o usuário autenticado pertence.
+ * **extra** - Um mapa de chaves de string arbitrárias para valores de string, fornecido pela camada de autenticação.
+ * **API** - Indica se a solicitação é para um recurso de API.
+ * **Caminho da requisição** - Caminho para diversos endpoints sem recursos, como `/api` ou `/healthz`.
+ * **Verbo de requisição de API** - Verbos da API como `get`, `list`, `create`, `update`, `patch`, `watch`, `delete` e `deletecollection` que são utilizados para solicitações de recursos. Para determinar o verbo de requisição para um endpoint de recurso de API , consulte [Determine o verbo da requisição](/pt-br/docs/reference/access-authn-authz/authorization/#determine-the-request-verb).
+ * **Verbo de requisição HTTP** - Métodos HTTP em letras minúsculas como `get`, `post`, `put` e `delete` que são utilizados para requisições que não são de recursos.
+ * **Recurso** - O identificador ou nome do recurso que está sendo acessado (somente para requisições de recursos) -- Para requisições de recursos usando os verbos `get`, `update`, `patch` e `delete`, deve-se fornecer o nome do recurso.
```

Comment:
`--` is not usually used in Portuguese, so we can substitute it for `-`. In this case, the following "Para" should start with a lowercase "p".